app/components/appeals-app-services/iam.tf

```diff
@@ -41,3 +41,9 @@ resource "azurerm_role_assignment" "appeals_fo_send_service_bus_role" {
   role_definition_name = "Azure Service Bus Data Sender"
   principal_id         = module.front_office_subscribers[0].principal_id
 }
+
+resource "azurerm_role_assignment" "appeals_fo_api_send_service_bus_role" {
+  scope                = var.back_office_service_bus_id
```

Comment:
Think it would be more secure to only give FO send access to specific topics rather than the entire service bus especially as it's a back office resource (least privilege)

Reply:
Yeah agreed, this is a bit of accepted tech debt for me to speed up development of other upcoming integrations so we don't need to work on infra every time in the short term in the same way BO are doing at the moment. Happy to refactor though.

Reply:
if we can get a ticket in to change it before we go to prod then happy to approve?

Reply:
I've ended up having to add the config changes so may as well just tweak this at the same time